monkey/monkey_island/cc/services/initialize.py

```diff
@@ -138,10 +138,16 @@ def _decorate_island_event_queue(
 
 
 def _register_repositories(container: DIContainer, data_dir: Path):
-    container.register_instance(
+    container.register_convention(
```

Comment:
This change isn't necessary. Conventions are a necessary evil. "Evil" is maybe a strong word; they're not preferable.

There's no need to force most components to call their `IFileRepository` a specific thing.